paasta_tools/kubernetes_tools.py

```diff
@@ -2049,3 +2099,25 @@ def get_pod_hostname(kube_client: KubeClient, pod: V1Pod) -> str:
         return pod.spec.node_name
     # if label has disappeared (say we changed it), default to node name
     return node.metadata.labels.get("yelp.com/hostname", pod.spec.node_name)
+
+
+def to_node_label(label: str) -> str:
```

Comment:
@EvanKrall can you weigh in on this function? Unlike before with mesos we just injected constraints kinda "as is".
With this function we are adding a bit of abstraction but converting things for users.